blackbox/monitoring/src/tests.py

```diff
@@ -134,57 +136,62 @@ def test_enterprise_setting_disabled(self):
 
 class MonitoringNodeStatusIntegrationTest(unittest.TestCase):
 
-    JMX_PORT = GLOBAL_PORT_POOL.get()
-    CRATE_HTTP_PORT = GLOBAL_PORT_POOL.get()
-
     def test_mbean_select_ready(self):
-        jmx_client = JmxTermClient(MonitoringIntegrationTest.JMX_PORT)
-        value, _ = jmx_client.query_jmx(
+        jmx_client = JmxTermClient(JMX_PORT)
+        value, exception = jmx_client.query_jmx(
             'io.crate.monitoring:type=NodeStatus',
             'Ready'
         )
-        value = bool(value)
-        if value is not True:
+
+        if exception:
+            raise AssertionError("Unable to get attribute NodeStatus.Ready")
+
+        value = str(value, 'utf-8').rstrip('\n')
+        if value != 'true':
             raise AssertionError("The mbean attribute has not produced the expected result. " +
-                                 "Expected: True, Got: {}".format(value))
+                                 "Expected: true, Got: {}".format(value))
 
 
 class MonitoringNodeInfoIntegrationTest(unittest.TestCase):
 
-    JMX_PORT = GLOBAL_PORT_POOL.get()
-    CRATE_HTTP_PORT = GLOBAL_PORT_POOL.get()
-
     def test_mbean_node_name(self):
-        jmx_client = JmxTermClient(MonitoringIntegrationTest.JMX_PORT)
-        nodeName, _ = jmx_client.query_jmx(
+        jmx_client = JmxTermClient(JMX_PORT)
+        nodeName, exception = jmx_client.query_jmx(
             'io.crate.monitoring:type=NodeInfo',
             'NodeName'
         )
 
-        if not nodeName:
+        if exception:
+            raise AssertionError("Unable to get attribute NodeInfo.NodeName")
+
+        nodeName = str(nodeName, 'utf-8').rstrip('\n')
+        if nodeName != 'crate-enterprise':
```

Comment:
the following error message should be adjusted (its not an empty string check anymore)